### PR TITLE
Enable output of script line for 'run' in verbose mode

### PR DESCRIPTION
**Pull request by refack**
### The issue

There is no simple way to see the what is the command line that's used for `pipenv run`

### The fix

Output the command line to the log as `info`, so it's output when run in verbose mode.

### The checklist

* [ ] Associated issue
* [x] A news fragment in the `news/` directory to describe this fix with the extension `.bugfix`, `.feature`, `.behavior`, `.doc`. `.vendor`. or `.trivial` (this will appear in the release changelog). Use semantic line breaks and name the file after the issue number or the PR #.

Seeking early feedback.

### CODE REVIEW

File: pipenv/core.py [hunk 1]
```diff
@@ -2303,6 +2303,8 @@ def do_run(command, args, three=None, python=False, pypi_mirror=None):
     inline_activate_virtual_environment()
     try:
         script = project.build_script(command, args)
+        cmd_string = ' '.join([script.command] + script.args)
+        logging.getLogger("pip").info("Run: '{0}'".format(cmd_string))
```
Comment (frostming) [inline pipenv/core.py]: This logger is not used in any other place of this file. To keep consistency, I recommend
```python
if environments.is_verbose():
    click.echo("Run: {!0r}".format(cmd_string), err=True)
```

Comment (refack) [reply pipenv/core.py]: Sure.

**Comment by jxltom** [comment]
```
➜  abc pipenv --verbose run pip list 
Run: pip list
Package    Version
---------- -------
pip        18.1   
setuptools 40.6.2 
wheel      0.32.3 
```

Here is the example output making easier to review

File: pipenv/core.py (hunk 1 above)
Comment (techalchemy) [reply pipenv/core.py]: (as a sidenote, logging using a logger implementation isn't a great practice, especially in our case -- we parse the output that goes to stdout in many cases)

File: pipenv/core.py [hunk 2]
```diff
@@ -2303,6 +2303,9 @@ def do_run(command, args, three=None, python=False, pypi_mirror=None):
     inline_activate_virtual_environment()
     try:
         script = project.build_script(command, args)
+        cmd_string = ' '.join([script.command] + script.args)
+        if environments.is_verbose():
+            click.echo(crayons.normal("Run: {0}".format(cmd_string)), err=True)
```
Comment (techalchemy) [inline pipenv/core.py]: I have one small request -- in order to be consistent across our output elsewhere, the ideal output format would be something to the effect of:
```suggestion
            click.echo(crayons.normal("$ {0}".format(cmd_string)), err=True)
```

**Comment by refack** [comment]
Another output example
For this `Pipfile`:
```TOML
[scripts]
lint = "flake8 . --exclude=.venv --count --show-source --statistics --select=E901,E999,F821,F822,F823"
```
Console:
```cmd
DEV D:\code\4node\gyp>pipenv -v run lint
Run: flake8 . --exclude=.venv --count --show-source --statistics --select=E901,E999,F821,F822,F823
0

DEV D:\code\4node\gyp>
```

**Comment by refack** [comment]
> can you please include a news fragment in the `news/`

Done, PTAL.

BTW: That's very crufty, could it be extracted from the commit log? Point me to the tools I'll give it a try :)

**Comment by techalchemy** [comment]
It's a pretty common changelog approach, we have a lot of commits and extracting changelogs from commit logs is not really a reasonable approach

**Pull request merged**